src/graphql/resolvers/box-resolver.ts

```diff
@@ -204,7 +231,8 @@ export class BoxResolver {
       boxIds,
       heightType,
       skip,
-      take
+      take,
+      orderBy: order
```

Comment:
Could you please confirm that you've tested different ordering options by running the code as well?